src/mscorlib/src/System/Collections/Generic/List.cs

```diff
@@ -976,8 +976,9 @@ public void Sort(IComparer<T> comparer)
             }
             Contract.EndContractBlock();
 
-            if( _size > 0) {
+            if (_size > 0) {
                 ArraySortHelper<T>.Sort(_items, 0, _size, comparison);
+                _version++;
```

Comment:
> is there a reason we sort on 1-item collections?

Not that I can think of.  Technically this would be an observable change, e.g. if the comparer throws, but seems like a fine thing to do to change the check to be > 1 rather than > 0.